app/build.gradle

```diff
@@ -28,11 +28,13 @@ dependencies {
     implementation 'androidx.appcompat:appcompat:1.1.0'
     implementation 'androidx.core:core-ktx:1.1.0'
     implementation 'androidx.constraintlayout:constraintlayout:1.1.3'
+    //implementation 'android.arch.navigation:navigation-fragment:1.0.0'
 
     implementation 'org.osmdroid:osmdroid-android:6.1.0'
 
     // Dagger
     implementation 'com.google.dagger:dagger:2.24'
+    implementation 'androidx.legacy:legacy-support-v4:1.0.0'
```

Comment:
What is this dependency for? Does it used anywhere in project? I've built application without it and it works same way.